app/views/two_factor_authentication/totp_verification/show.html.slim

```diff
@@ -1,10 +1,8 @@
 - title t('titles.enter_2fa_code')
 
 
-h1.h3.my0 = t('devise.two_factor_authentication.totp_header_text')
-p.mt-tiny.mb0#code-instructs = t('instructions.2fa.totp_intro_html', \
-      email: current_user.email,
-      app: APP_NAME)
+h1.h3.my0 = @presenter.header
+p.mt-tiny.mb0#code-instructs == @presenter.help_text
```

Comment:
Ditto for the `==` here -- if we rename the corresponding i18n keys to end with `html` then we can remove this `==` and turn it into `=`:

so `instructions.2fa.{sms,voice,totp}.confirm_code` ➡️ `instructions.2fa.{sms,voice,totp}.confirm_code_html` for example